docs/contributing.md

```diff
@@ -10,35 +10,36 @@ The following is a set of rules for contributing to **Rucio** and its
 packages. Use your best judgment, and feel free to propose changes to this
 document.
 
-If you have questions, you can reach the core development team on our 
-[__Mattermost__](mattermost.md) channel, or send an email to our
-development mailing list [__rucio-dev@cern.ch__](mailto:rucio-dev@cern.ch).
+If you have questions, you can reach the development team on our 
+[__Mattermost__](mattermost.md) channel.
 
 ## What should I know before I get started
 
-A contribution can be either be a **patch** or **feature**:
+A contribution can be either be included in the **next** major release or the
+ongoing **current** release line, e.g. `release-33` This becomes relevant when 
+naming the branches of your contributions (See section 3) so the Rucio maintainers 
+can understand the targeted release. 
 
-* **Patches** include bugfixes and minor changes to the code and are included in
-  patch releases usually made on a bi-weekly schedule.
-* **Features** include major developments or potentially disruptive changes and
-  are included in feature releases made multiple times a year.
+* Larger feature developments, backwards-compatibility breaking changes, changes
+  to the database schema, significant changes to the REST interface, and similar
+  **MUST** be targeted to the **next** release line.
+* Bugfixes, smaller backwards-compatbile changes to the REST interface, smaller
+  feature developments, and similar **CAN** be targeted to the **current** release
+  line.
 
 The [__repository__](https://github.com/rucio/rucio/) consists of different
 branches:
 
-* the **master** branch includes the development for the next major version.
-* the **release-…** branches include the patch/minor development of the
-  releases.
-
-Release branches only exist for the currently maintained release
-versions. Hotfix branches are created on demand. Please communicate to the Rucio
-maintainers, if you wish to hotfix a previous release.
+* the **master** branch includes the developments for the next major release.
+* the **release-…** branches include the patch/minor developments of the
+  releases (Such as current and LTS releases). Release branches only exist for 
+  the currently maintained releases (See the [release policy](started/releasepolicy.md))
 
 Generally all [__pull requests__](https://github.com/rucio/rucio/pulls) are to
-be created against the Rucio **master** branch. Features will end up in the
-upstream **master** only and patches are cherry-picked to the maintained
-releases if applicable. Release-specific changes are excluded from that rule and
-might be needed if e.g. cherry-picking to the last release was not successful.
+be created against the Rucio **master** branch. Pull requests named/targeted for
+the **next** major release will end up in the upstream **master** only. Pull 
+requests named/targeted for the **current** release will be additionally 
```

Comment:
remove `additionally` and add `by the maintainer` after `cherry-picked`.

`branch` -> `branches` (could be more than one, right?)

Reply:
Yes could be. Will change.
I purposely did not add the whole LTS backport workflow here, since it's such a rare occurence and just complicates this document for no reason; The "average" developer doesn't really need to know this.